tslint.json

```diff
@@ -12,6 +12,7 @@
                 "static-method-regex": "^[a-z][\\w\\d]+$"
             }
         ],
-        "no-banned-terms": false
+        "no-banned-terms": false,
+        "prefer-type-cast": false
```

Comment:
I'm okay with deciding we want to prefer `myvar as MyType` syntax to `<MyType>myvar` syntax for consistency with our other projects, but it'd be nice to standardize on one or the other rather than allowing both; https://github.com/Microsoft/tslint-microsoft-contrib suggests that we could do so by enabling the `no-angle-bracket-type-assertion` alongside the disabling of `prefer-type-cast`